storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -189,6 +189,12 @@ xb_mysql_connect()
 	xb_mysql_query(connection, "SET NAMES utf8",
 		       false, true);
 
+	if (xb_mysql_numrows(connection, "SELECT @@wsrep_sync_wait", false) >
+				0) {
+		xb_mysql_query(connection, "SET SESSION wsrep_sync_wait=0", false,
+				true);
+	}
+
 	return(connection);
 }
```

Comment:
@rahulmalik87 can you please check against upstream? would this log an error message in logs?
upstream doesn't have wsrep_sync_wait

Reply:
Hello @satya-bodapati, no SELECT @@nonexist_Variables doesn't throw any error message in logs.